dask/dataframe/io/tests/test_parquet.py

```diff
@@ -106,14 +121,20 @@ def test_series(fn):
     assert_eq(df.x, ddf)
 
 
-def test_names(fn):
-    assert set(read_parquet(fn).dask) == set(read_parquet(fn).dask)
-    assert (set(read_parquet(fn).dask) !=
-            set(read_parquet(fn, columns=['x']).dask))
-    assert (set(read_parquet(fn, columns='x').dask) !=
-            set(read_parquet(fn, columns=['x']).dask))
-    assert (set(read_parquet(fn, columns=('x',)).dask) ==
-            set(read_parquet(fn, columns=['x']).dask))
+def test_names(fn, engine):
+    def read(fn):
+        return read_parquet(fn, engine=engine)
+
+    assert (set(read(fn).dask) == set(read(fn).dask))
+
+    assert (set(read(fn).dask) !=
+            set(read(fn, columns=['x']).dask))
+
+    assert (set(read(fn, columns='x').dask) !=
```

Comment:
I'm surprised if this works - probably because the string is iterable (i.e., would fail if the column name was not one-character long).